src/components/molecules/NavBar/NavBar.scss

```diff
@@ -150,55 +162,37 @@ $border-radius: 28px;
     }
   }
 }
-.nav-party-logo {
-  position: relative;
-  background-color: rgba($white, 0.08);
-  height: 60px;
-  padding: 0 50px 0 20px;
-  text-align: center;
-  cursor: pointer;
+
+.nav-schedule {
+  position: fixed;
   display: flex;
-  justify-content: space-between;
+  flex-direction: column;
   align-items: center;
-  overflow: hidden;
-  white-space: nowrap;
-  text-overflow: ellipsis;
-  transition: transform 400ms $transition-function;
-  border: none;
-  appearance: none;
+  justify-content: center;
+  top: 65px;
+  left: -$spacing--md;
   color: $white;
+  width: $nav-container-width;
+  height: $nav-container-height;
+  border-top-right-radius: 10px;
+  border-bottom-right-radius: 10px;
+  cursor: pointer;
+  background-image: url(/assets/icons/nav-schedule-whatson.png);
 
-  img {
-    height: 40px;
-  }
-  &:hover,
-  &:focus {
-    border: none;
-    background-color: rgba($white, 0.14);
-    &:after {
-      opacity: 0.7;
-    }
-  }
-  &:after {
-    content: "";
-    position: absolute;
-    right: 20px;
-    top: 20px;
-    height: 20px;
-    width: 20px;
-    background-image: url(/assets/icons/nav-party-logo-icon.png);
-    background-size: 20px;
-    opacity: 0.4;
-    transform: rotate(0deg);
-    transition: transform 400ms $transition-function;
+  &-clicked {
+    margin-left: 41%;
+    width: 30px;
+    height: $nav-btn-opened-height;
+    background-image: url(/assets/icons/nav-schedule-whatson-clicked.png);
```

Comment:
please, remove the images you don't need anymore. 🙃

Reply:
please, use a transparent picture instead of the ones with the background. and change the background color in the css.
![image](https://user-images.githubusercontent.com/63313732/130686572-62df5583-20be-410f-aa6a-a7a247d8ba14.png)

Reply:
fixed in e1ea69a
removed icons as well